### PR TITLE
adding 4th car

### PR DESCRIPTION
**Pull request by elnesh**

### CODE REVIEW

File: GitHubTraining/Program.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@ static void Main(string[] args)
             cars.Add(new Car("Mazda", 2016, "Blue"));
             cars.Add(new Car("Nissan", 2010, "Red"));
             cars.Add(new Car("Bentley", 1950, "Red"));
+            cars.Add(new Car("Toyota", 2012, "White"));
```
Comment (nashneshev) [inline GitHubTraining/Program.cs:33]: Looks good, but consider adding one more car.

**Review by nashneshev — APPROVED** [review]

File: GitHubTraining/Program.cs (hunk 1 above)
Comment (elnesh) [reply GitHubTraining/Program.cs:33]: ok, I will add it.

File: GitHubTraining/Program.cs [hunk 2]
```diff
@@ -31,6 +31,7 @@ static void Main(string[] args)
             cars.Add(new Car("Nissan", 2010, "Red"));
             cars.Add(new Car("Bentley", 1950, "Red"));
             cars.Add(new Car("Toyota", 2012, "White"));
+            cars.Add(new Car("Bugati", 1980, "Metalic"));
```
Comment (nashneshev) [inline GitHubTraining/Program.cs]: Great, thanks! Feel free to complete the request.

Comment (elnesh) [reply GitHubTraining/Program.cs]: Great!

**Pull request merged**